tests/integration/test_charm.py

```diff
@@ -33,9 +33,6 @@ async def test_build_and_deploy(ops_test: OpsTest):
           {mc.name}:
             charm: {charm}
             trust: true
-            resources:
```

Comment:
were these changes necessary?

Reply:
Yes. resources "key" doesn't exist in this current charmcraft. it breaks the test.